pygments/lexers/apdlexer.py

```diff
@@ -0,0 +1,211 @@
+    
+    # list of in-built () functions
+    elafunf = ["NX()","NY()","NZ()","KX()","KY()","KZ()","LX()","LY()","LZ()","LSX()","LSY()","LSZ()","NODE()","KP()","DISTND()","DISTKP()","DISTEN()","ANGLEN()","ANGLEK()","NNEAR()","KNEAR()","ENEARN()","AREAND()","AREAKP()","ARNODE()","NORMNX()","NORMNY()","NORMNZ()","NORMKX()","NORMKY()","NORMKZ()","ENEXTN()","NELEM()","NODEDOF()","ELADJ()","NDFACE()","NMFACE()","ARFACE()","UX()","UY()","UZ()","ROTX()","ROTY()","ROTZ()","TEMP()","PRES()","VX()","VY()","VZ()","ENKE()","ENDS()","VOLT()","MAG()","AX()","AY()","AZ()","VIRTINQR()","KWGET()","VALCHR()","VALHEX()","CHRHEX()","STRFILL()","STRCOMP()","STRPOS()","STRLENG()","UPCASE()","LWCASE()","JOIN()","SPLIT()","ABS()","SIGN()","CXABS()","EXP()","LOG()","LOG10()","SQRT()","NINT()","MOD()","RAND()","GDIS()","SIN()","COS()","TAN()","SINH()","COSH()","TANH()","ASIN()","ACOS()","ATAN()","ATAN2()"]
+    elafung = ["NSEL()","ESEL()","KSEL()","LSEL()","ASEL()","VSEL()","NDNEXT()","ELNEXT()","KPNEXT()","LSNEXT()","ARNEXT()","VLNEXT()","CENTRX()","CENTRY()","CENTRZ()"]
+    elafunh = ["~CAT5IN","~CATIAIN","~PARAIN","~PROEIN","~SATIN","~UGIN","A","AADD","AATT","ABEXTRACT","\*ABBR","ABBRES","ABBSAV","ABS","ACCAT","ACCOPTION","ACEL","ACLEAR","ADAMS","ADAPT","ADD","ADDAM","ADELE","ADGL","ADRAG","AESIZE","AFILLT","AFLIST","AFSURF","\*AFUN","AGEN","AGLUE","AINA","AINP","AINV","AL","ALIST","ALLSEL","ALPHAD","AMAP","AMESH","/AN3D","ANCNTR","ANCUT","ANCYC","ANDATA","ANDSCL","ANDYNA","/ANFILE","ANFLOW","/ANGLE","ANHARM","ANIM","ANISOS","ANMODE","ANMRES","/ANNOT","ANORM","ANPRES","ANSOL","ANSTOAQWA","ANSTOASAS","ANTIME","ANTYPE"]
+
+    tokens = {
+        'root': [
+            (r'!.*\n', Comment),
+            include('strings'),
+            ("(" + "|".join(elafunb) + r')\b',  Name.Class),
```

Comment:
This should use the `words` function

Reply:
Thanks. Can you give me more information? I am unsure on how to add/use this function.

Reply:
Following [these](https://pygments.org/docs/lexerdevelopment/?highlight=words#handling-lists-of-keywords) instructions I've joined all the keywords lists with words, but I am unsure if I did it the right way. I am still a newbie at Python and I did this quite a long time ago, so I can't remember what I wrote very well... Thanks for any help